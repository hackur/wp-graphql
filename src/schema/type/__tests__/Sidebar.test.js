import SidebarType from 'type/Sidebar';

describe('Test Sidebar type', () => {
  test('Test name', () => {
    expect(SidebarType.name).toMatchSnapshot();
  });

  test('Test fields', () => {
    expect(Object.keys(SidebarType.getFields())).toMatchSnapshot();
  });
});
