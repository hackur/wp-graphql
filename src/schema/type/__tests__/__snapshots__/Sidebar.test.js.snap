// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Sidebar type Test fields 1`] = `
Array [
  "id",
  "name",
  "description",
  "classname",
  "before_widget",
  "after_widget",
  "before_title",
  "after_title",
  "widgets",
]
`;

exports[`Test Sidebar type Test name 1`] = `"Sidebar"`;
